api/organisations/subscriptions/metadata.py

```diff
@@ -9,38 +11,44 @@ def __init__(
         seats: int = 0,
         api_calls: int = 0,
         projects: typing.Optional[int] = None,
-        chargebee_email=None,
+        chargebee_email: str = None,
+        audit_log_visibility_days: int | None = 0,
+        feature_history_visibility_days: int | None = DEFAULT_VERSION_LIMIT_DAYS,
     ):
         self.seats = seats
         self.api_calls = api_calls
         self.projects = projects
         self.chargebee_email = chargebee_email
+        self.audit_log_visibility_days = audit_log_visibility_days
+        self.feature_history_visibility_days = feature_history_visibility_days
 
     def __add__(self, other: "BaseSubscriptionMetadata"):
         if self.payment_source != other.payment_source:
             raise TypeError(
                 "Cannot add SubscriptionMetadata from multiple payment sources."
             )
 
-        if self.projects is not None and other.projects is not None:
-            projects = self.projects + other.projects
-        elif self.projects is None and other.projects is not None:
-            projects = other.projects
-        elif other.projects is None and self.projects is not None:
-            projects = self.projects
-        else:
-            projects = None
-
         return self.__class__(
             seats=self.seats + other.seats,
             api_calls=self.api_calls + other.api_calls,
-            projects=projects,
+            projects=_add_nullable_limit_attributes(self.projects, other.projects),
```

Comment:
Note: this does technically change the behaviour since, previously, if we added 2 metadata objects together where one had `project=None` and the other had `projects=1` the result would be `projects=1` but (a) this didn't make sense because `None` implied no limit so adding 1 should have kept the 'no limit', and (b) we don't actually use this functionality because we don't have any add ons for extra projects (certainly not that we use in SaaS).